Sources/SmithyHTTPAPI/SdkHttpRequest.swift

```diff
@@ -118,6 +118,19 @@ extension SdkHttpRequest: CustomDebugStringConvertible, CustomStringConvertible
         description
     }
 
+    public var debugDescriptionWithoutAuthorizationHeader: String {
+        let method = method.rawValue.uppercased()
+        let protocolType = self.destination.scheme
+        let query = self.destination.queryString ?? ""
+        let port = self.destination.port.map { String($0) } ?? ""
+        let header = headers.dictionary
+            .filter({ $0.key != "Authorization" })
+            .map({"\($0.key): \($0.value.joined(separator: ", "))"})
+            .joined(separator: ", \n")
```

Comment:
nit: for readability use named variables instead of `$0`

Example:
```
let header = headers.dictionary
    .filter { key, _ in key != "Authorization" }
    .map { key, value in "\(key): \(value.joined(separator: ", "))" }
    .joined(separator: ", \n")
```

Reply:
Changed as suggested, after discussion.